app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -80,7 +80,7 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="12dp"
             android:text="@string/admin_auth_sub"
-            android:textColor="@color/oppia_admin_auth_secondary_text"
+            android:textColor="@color/admin_auth_secondary_text"
```

Comment:
One major suggestion. I think all those colors which are defined in `component_colors.xml` should have a end suffix of `_color` and everything inside `color_palette` and `color_defs` should not have suffix `_color` this way we can easily add a regex for all three files to clearly follow above rules and also it will set a good coding convention.

Reply:
@BenHenning Any thoughts on this?

Reply:
Basically we need a convention to make sure when to use `_color` as suffix and when not too.

Reply:
I think that makes sense for color_defs and component_colors, but how would it work for color_palette? Could you maybe add some examples to demonstrate the convention?

Reply:
Actually you are right. @BenHenning 
We can need `_color` as suffix for all values in `color_palette`.
So lets follow this

`color_defs` -> does not contain `_color` as in this file we are mentioning colors like `black`, `white` which by default indicate color.
`color_palette` -> all values contain `_color` as suffix
`color_components` -> all values contain `_color` as suffix


@BenHenning Any thoughts?

Reply:
This convention sounds good to me.

Reply:
That SGTM @rt4914.